functions.inc.php

```diff
@@ -1918,4 +1918,21 @@ function getRemoteAddr() {
     return $REMOTE_ADDR;
 }
 
+
+/**
+ * Returns a hash for a username valid for one day
+ *
+ * @param String $username user name
+ * @return String password recovery code
+ */
+function getPasswordRecoveryCode($username)
+{
+    $username = trim(strtolower($username));
+    $date = date('Y-m-d');
+
+    $code = substr(strtoupper(md5('SECRET SALTING PHRASE' . $username . $date)), 0, 6);
```

Comment:
That would be better than not including any secret at all, but
- the setup password might be empty (you can create your first admin with postfixadmin-cli, so you don't really need a setup password) - so don't rely on it
- re-using the same secret every time could allow to guess/brute force it if a user uses "lost password" several times
- this patch is security relevant, so it MUST be secure. Not "somewhat secure".